mmv1/third_party/terraform/tests/resource_compute_router_bgp_peer_test.go

```diff
@@ -675,3 +718,100 @@ resource "google_compute_router_peer" "foobar" {
 }
 `, routerName, routerName, routerName, routerName, routerName, routerName, routerName, routerName, routerName, routerName, routerName, enable)
 }
+
+func testAccComputeRouterPeerBfd(routerName, bfdMode string) string {
```

Comment:
What stops us from making this the example? It's a lot of config but if that's required to use the feature, I don't see a problem documenting it! See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_service_attachment#example-usage---service-attachment-basic for example

Reply:
BFD doesn't make much sense for VPN right now as it can only be `DISABLED`. A more realistic example is to use it with interconnect but that can't be tested due to the APIs. I'm OK making this a proper example once BFD support for VPN is added, though.

Reply:
SGTM, thanks!